### DISCUSSION TITLE
Fly pad 3 gets Block fuel not Min takeoff

### DISCUSSION THREAD

**Discussion by Swaluver88** [flybywiresim/aircraft#6975] [Help]
hello, 
Today I noticed that the FLYPAD3 take the block fuel from simbrief and not the Min TO and or MAX TO fuel. I would think it should grab at least the Min TO fuel. 

![Microsoft Flight Simulator Screenshot 2022 03 27 - 18 14 40 19](https://user-images.githubusercontent.com/84532990/160310808-2921c07d-ee5d-4226-a54b-3ac8a7ed7cff.png)
![20220327_180943](https://user-images.githubusercontent.com/84532990/160310837-a0e371fc-1444-49a9-aa7d-abea118a7600.jpg)

is this planned to be fixed?

**Comment by tracernz** [comment]
Something is wrong with your OFP here... takeoff fuel (and consequently block fuel) is less than your trip fuel... Why is extra negative?

To borrow from Skybrary:
> Block fuel is the total fuel required for the flight and is the sum of the Taxi fuel, the Trip fuel, the Contingency fuel, the Alternate fuel, the Final Reserve fuel, the Additional fuel and any Extra fuel carried.

Seems like the correct one to me?

**Reply by Swaluver88** [reply]
Where are you seeing that? 
I'm seeing min TO fuel is 24536 and Block is 19045 and trip is 19260
I'm gonna do a couple short hops today, I just know, due to what I actually do for work, you have to take the min require TO fuel unless your have tanking fuel then you can take either MIN TO -> MAX TO if it doesn't exceed ATOG and or MZFW. 

Also trip fuel is 19260 so there is no way that block fuel definition stated above is correct in that aspect.

**Reply by donstim** [reply]
You cannot take the minimum required takeoff fuel if it exceeds the amount of fuel the airplane can carry.

**Comment by donstim** [comment]
You have selected a route and payload that is beyond the payload-range capability of the A230NEO we are simulating. The maximum amount of usable fuel you can carry is 19,046 kg. For some reason, if the flight requires more fuel than the airplane can carry, simBrief shows the overage as a negative amount of extra fuel.

**Reply by Swaluver88** [reply]
Yeah I set my payload to Zero in simbrief, don't know why simbrief still put a payload in there, I didn't great the W&B from AOC left it at zero

Gonna do a couple more shorter hauls and see if the same problem persists

**Reply by donstim** [reply]
The "problem" is not with the airplane. The route you chose is too long at simBrief's estimated fuel burn. You cannot carry enough fuel for that route.

**Comment by CptBayer** [comment]
the V3 pad is experminetal isn´t it? I flew a leg today with dev version and I was wondering after importing the OFP via simbrief and fueling, that the GW was about 2 tons to low in comparision with the flightplan. I didn´t investigate due to lack of time, but next leg I will check if it is similar to todays leg ..

greets

**Reply by donstim** [reply]
When you do, please open a new discussion as that is a different issue. Make sure that you are using the current Simbrief profile per the documentation. If you are using weights in pounds, we are aware that there can be a discrepancy in the weights. We are investigating.

**Comment by CptBayer** [comment]
the V3 pad is experminetal isn´t it? I flew a leg today with dev version and I was wondering after importing the OFP via simbrief and fueling, that the GW was about 2 tons to low in comparision with the flightplan. I didn´t investigate due to lack of time, but next leg I will check if it is similar to todays leg ..

greets

**Reply by tracernz** [reply]
Make sure to update your simbrief "airframe" as donstim said. https://docs.flybywiresim.com/fbw-a32nx/installation/#simbrief-airframe
